lib/seattleflu/db/cli/command/clinical.py

```diff
@@ -191,7 +191,7 @@ def missing_barcode(df: pd.DataFrame) -> pd.DataFrame:
     missing_barcodes = df.loc[df['barcode'].isnull()].copy()
     missing_barcodes['problem'] = 'Missing barcode'
 
-    return missing_barcodes[['MRN', 'Accession', 'barcode', 'problem', '_metadata']]
+    return missing_barcodes
```

Comment:
Can you explain the reasoning for removing the subseting here (and in `duplicated_barcode` below)? My guess is that "more is better" since the full data for each row is only going to a file now?

Reply:
Good question! The original subsetting was arbitrary. It was done for the purposes of printing a smaller table to stdout. Now that we're writing de-identified log messages -or- generating a new "problem barcodes" csv, there is no need for subsetting.